rclpy/actions/minimal_action_client/client_not_composable.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from example_interfaces.action import Fibonacci
+
+import rclpy
+from rclpy.action import ActionClient
+from rclpy.node import Node
+
+
+def feedback_cb(logger, feedback):
+    logger.info('got feedback {0}'.format(feedback))
+
+
+def main(args=None):
+    rclpy.init(args=args)
+
+    node = rclpy.create_node('minimal_action_client')
+
+    action_client = ActionClient(node, Fibonacci, 'fibonacci')
+
+    node.get_logger().info('Waiting for action server...')
+    action_client.wait_for_server()
+
+    goal_msg = Fibonacci.Goal()
+    goal_msg.order = 10
+
+    send_goal_future = action_client.send_goal_async(
+        goal_msg, feedback_callback=lambda feedback: feedback_cb(node.get_logger(), feedback))
+
+    rclpy.spin_until_future_complete(node, send_goal_future)
```

Comment:
Currently seeing an assertion while spinning here
```
ros2 run examples_rclcpp_minimal_action_server action_server_not_composable
```

```
$ ros2 run examples_rclpy_minimal_action_client client_not_composable
[INFO] [minimal_action_client]: Waiting for action server...
python3: /home/sloretz/ws/ros2/src/ros2/rclpy/rclpy/src/rclpy/_rclpy_action.c:850: rclpy_action_take_status: Assertion `create_ros_message != NULL && "unable to retrieve create_ros_message function, type_support must not have been imported"' failed.
```

Reply:
Yeah, I noticed that. Looking into it.
With a mock action server in Python, things seem to be okay.

Reply:
The issue is related to missing type support for the goal status array message. I'll open a PR with a fix.